utils/HistorySaver.tsx

```diff
@@ -20,6 +20,7 @@ export type HistoryType = {
     };
     time: string;
     status: string | number;
+    authMethod: string;
```

Comment:
`authMethod` will contain authentication method (eg: *basic*). This will be passed into `authslide` to select the right section